src/rpc/mining.cpp

```diff
@@ -961,6 +1004,7 @@ static const CRPCCommand commands[] =
 
 
     { "generating",         "generatetoaddress",      &generatetoaddress,      {"nblocks","address","maxtries"} },
+    { "generating",         "generatetodescriptor",   &generatetodescriptor,   {"num_blocks","descriptor","maxtries"} },
```

Comment:
As it is test-only, it should probably be hidden from the normal help.

Reply:
`generatetoaddress` isn't hidden, so I used the same. Otherwise it would also be missing from the gui-autocomplete.